src/utils/ocean.ts

```diff
@@ -0,0 +1,87 @@
+import { ConfigHelper, ConfigHelperConfig, Logger } from '@oceanprotocol/lib'
+import contractAddresses from '@oceanprotocol/contracts/artifacts/address.json'
+import { AbiItem } from 'web3-utils/types'
+import Web3 from 'web3'
+import { chains } from '../../chains.config'
+import { ConfigHelperConfigOverwrite } from '../@types/Chains'
+
+export function getOceanConfig(network: string | number): ConfigHelperConfig {
+  const config = new ConfigHelper().getConfig(
+    network,
+    network === 'moonbeamalpha' ||
+      network === 1287 ||
+      network === 'bsc' ||
+      network === 56 ||
```

Comment:
thought(non-blocking): as we only support gx chain for now these can probably be removed safely.